sympy/stats/tests/test_continuous_rv.py

```diff
@@ -795,22 +859,18 @@ def test_input_value_assertions():
         fn('x', p, q)  # No error raised
 
 
-@XFAIL
 def test_unevaluated():
     X = Normal('x', 0, 1)
-    assert E(X, evaluate=False) == (
-        Integral(sqrt(2)*x*exp(-x**2/2)/(2*sqrt(pi)), (x, -oo, oo)))
+    assert str(E(X, evaluate=False)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
+    "(2*sqrt(pi)), (x, -oo, oo))")
 
-    assert E(X + 1, evaluate=False) == (
-        Integral(sqrt(2)*x*exp(-x**2/2)/(2*sqrt(pi)), (x, -oo, oo)) + 1)
+    assert str(E(X + 1, evaluate=False)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
+    "(2*sqrt(pi)), (x, -oo, oo)) + 1")
 
-    assert P(X > 0, evaluate=False) == (
-        Integral(sqrt(2)*exp(-x**2/2)/(2*sqrt(pi)), (x, 0, oo)))
+    assert str(P(X > 0, evaluate=False)) == ("Integral(sqrt(2)*exp(-_z**2/2)/"
```

Comment:
What's the output without using `str`?

Reply:
Same, but it does not evaluate to true while testing, the reason being the use of dummy `_z`.  @sidhantnagpal suggested this [here](https://github.com/sympy/sympy/pull/16827#discussion_r283788020).